gprofiler/main.py

```diff
@@ -572,6 +576,14 @@ def parse_cmd_args() -> configargparse.Namespace:
         help="Whether to upload the profiling results to the server",
     )
 
+    parser.add_argument(
+        "--dont-disable-core-files",
+        action="store_false",
+        dest="disable_core_files",
+        help="Do not disable creation of coredumps for processes started by the profiler. By default, we disable,"
+        " to refrain from generating core files which consume disk space, in case any of the executed profilers crash.",
+    )
```

Comment:
where is the default?

Reply:
If you use `store_false` then it defaults to True